tool/release/deps/rules.bzl

```diff
@@ -48,7 +48,7 @@ _release_validate_deps_script_test = rule(
         ),
```

Comment:
Update packages and Bazel targets which we forgot to update in https://github.com/vaticle/dependencies/pull/327.